torchvision/datasets/imagenet.py

```diff
@@ -21,6 +21,10 @@
 class ImageNet(ImageFolder):
     """`ImageNet <http://image-net.org/>`_ 2012 Classification Dataset.
 
+    Before using this class, it is required to download `ImageNet 2012` dataset from
```

Comment:
Can we be more specific about what files are needed here? We always need the devkit `ILSVRC2012_devkit_t12.tar.gz` and `ILSVRC2012_img_train.tar` or `ILSVRC2012_img_val.tar` based on `split`.

Reply:
Thanks, I added.